src/components/table/OcTableFiles.vue

```diff
@@ -711,64 +711,210 @@ export default {
       selected: []
     }),
     computed: {
-      resources() {
-        return [
-          {
-            name: "forest.jpg",
-            path: "images/nature/forest.jpg",
-            thumbnail: "https://cdn.pixabay.com/photo/2015/09/09/16/05/forest-931706_960_720.jpg",
-            indicators: [],
-            type: "file",
-            sdate: "Mon, 11 Jan 2021 14:34:04 GMT",
-            owner: [{
+          groupingSettings(){
+      return {
+        groupingAllowed: true,
+        defaultGroupingBy: "None",
+        passedGroupingBy: "",  //None, creation, owner,alphabetically
+        showGroupingOptions: true,
+        previewTable: true,
```

Comment:
Same for here: can't we use `previewAmount` only? If it's 0, is off, it's it's not 0, then we take it.

Reply:
good point